tests/excerptexport/management/test_purge_old_result_files.py

```diff
@@ -24,6 +24,21 @@ def output_file_with_file_too_old(db, output_file_with_file, mocker):
     return output_file_with_file
 
 
+@pytest.fixture
+def output_file_too_old(db, output_file, mocker):
+    from osmaxx.excerptexport._settings import RESULT_FILE_AVAILABILITY_DURATION
+    just_a_bit_too_old = timezone.now() - RESULT_FILE_AVAILABILITY_DURATION - timezone.timedelta(minutes=1)
+
+    output_file.file_removal_at = just_a_bit_too_old
```

Comment:
```python
output_file.file_removal_at = timezone.now() - timezone.timedelta(minutes=1)
```
should suffice for the file being expired. Why also subtract `RESULT_FILE_AVAILABILITY_DURATION`?